package.json

```diff
@@ -20,8 +24,8 @@
   "scripts": {
     "start": "parcel src/index.html",
     "test": "jest --coverage",
-    "lint": "eslint src/*.js src/*.jsx",
-    "lint-fix": "eslint --fix src/*.js src/*.jsx"
+    "lint": "eslint src/*/*.js src/*/*.jsx",
```

Comment:
Ah I just found you can run eslint recursively like this:

eslint **/*.js 

Try that.